src/index.html

```diff
@@ -1,14 +1,55 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id = "home">
+    <header class="header">
+      <a href="#home" class="header__logo-link">
+        <img
+          src="./images/logo.png"
+          alt="header logo image"
+          class="header__logo">
```

Comment:
1. I would suggest that `MOYO logo` makes more sense than `header logo image` for the search robots and screen readers.
2. Correct formatting would be:
```
<img
  src="./images/logo.png"
  alt="MOYO header"
  class="header__logo"
>
```

Reply:
done